obspy/io/sac/header.py

```diff
@@ -16,22 +16,22 @@
 
 # header documentation is large and used in several places, so we just write
 # it once here and distributed it as needed.
-DOC = {'npts': 'N    Number of points per data component. [required]',
+DOC = {'npts': 'N    Number of points per data component.',
        'nvhdr': '''N    Header version number. Current value is the integer 6.
                   Older version data (NVHDR < 6) are automatically updated
-                  when read into sac. [required]''',
-       'b': 'F    Beginning value of the independent variable. [required]',
-       'e': 'F    Ending value of the independent variable. [required]',
-       'iftype': '''I    Type of file [required]:
+                  when read into sac.''',
+       'b': 'F    Beginning value of the independent variable.',
+       'e': 'F    Ending value of the independent variable.',
+       'iftype': '''I    Type of file:
 
                   * ITIME {Time series file}
                   * IRLIM {Spectral file---real and imaginary}
                   * IAMPH {Spectral file---amplitude and phase}
                   * IXY {General x versus y data}
                   * IXYZ {General XYZ (3-D) file}''',
-       'leven': 'L    TRUE if data is evenly spaced. [required]',
+       'leven': 'L    TRUE if data is evenly spaced.',
        'delta': 'F    Increment between evenly spaced samples (nominal value).'
-                ' [required]',
+                '',
```

Comment:
If you want to fit more on a line, there is a way, but it means changing everything in the dictionary. Instead of aligning to the opening `{`, put the first entry on a new line and indent by 4 spaces.